### ISSUE TITLE
Integration with Flexget

### ISSUE THREAD

**Issue by louist87** [louist87/Scrappy#6]
**The Purpose**

Scrappy should expose an interface to act as a plugin with [Flexget](http://flexget.com/), thereby allowing Flexget users to automatically rename new files based on TVDB information.  This is useful for allowing metadata scrapers (such as those found in [XBMC](http://xbmc.org/)) to find the correct information.

**The Point**
Flexget already offers rudimentary options for renaming files to a specific format, but several notable features are lacking:
- Information is drawn from filename and the parameters of the config file.  Incorrect episode names, for example, will not be fixed.
- Flexget users often specify TVDB ids in their configuration file, which means that Scrappy can have perfect performance in those cases.

**tl;dr:** This would do what flexget already does, but would catch a few corner cases that the former currently misses.

**Comment by louist87** [comment]
http://flexget.com/wiki/Developers

**Comment by rollbackpt** [comment]
how is the integration with flexget going?

**Comment by louist87** [comment]
It's coming along fairly well.  There's not much to it, but for some reason the test is failing.  Once I get that sorted out, I should be ready to issue a pull request.

**Comment by rollbackpt** [comment]
nice i am waiting to change my flexget config :D
